tests/unit/lib/build_module/test_app_builder.py

```diff
@@ -194,7 +195,7 @@ def test_must_use_lambda_builder(self, lambda_builder_mock):
                                                          "artifacts_dir",
                                                          "scratch_dir",
                                                          "manifest_path",
-                                                         "runtime")
+                                                         "runtime",)
```

Comment:
mode here as well?

Reply:
`mode` isn't passed into the `_build_function_in_process` function because `mode` is an attribute on the class (accessed through `self.mode`).